troubleshooting/known_issues/parry_peak_console_boot_errors.md

```diff
@@ -0,0 +1,77 @@
+# HPE Cray `EX255a` Boot Issue with Console Parameter
+
+- [Description](#description)
+- [Workaround](#workaround)
+
+## Description
+
+  `HPE Cray EX255a` hardware has an issue in it's firmware configuration that causes boots to stall when configured to use the default kernel serial port `ttyS0` which is provided in the compute image created during installation of CSM 1.5.2.
+
+## Workaround
+
+1. (`ncn-mw#`) Follow the steps for [customizing](../../operations/image_management/Customize_an_Image_Root_Using_IMS.md) an image to be used for booting `HPE Cray EX255a` hardware.
+
+    ```bash
+    cray ims jobs create --public-key-id $PK_ID --job-type customize --artifact-id $IMAGE_ID --image-root-archive-name $NEW_IMAGE_NAME
+    ```
+
+    Example output:
+
+    ```toml
+    artifact_id = "fdc16942-ba1d-49b5-a4a2-0c7bbce8bb68"
+    job_type = "customize"
+    image_root_archive_name = "uss-compute-jsmit-test"
+    status = "creating"
+    require_dkms = false
+    job_mem_size = 8
+    public_key_id = "b815341b-d096-47fa-aa92-11a673b9bb69"
```

Comment:
We probably don't care about public keys being in the documentation because they're supposed to be 'public', but I'll caution you about putting secrets of any kind in documentation. I've had to redact stuff like this before. You could just put 
public_key_id = <public_key_id>